src/addresses/Signature.ts

```diff
@@ -0,0 +1,41 @@
+import {Wallet} from "..";
+const bchMessage = require('bitcoinjs-message');
```

Comment:
I believe we already have methods to sign and verify using secp256k1 imported

https://libauth.org/interfaces/secp256k1.html

Although it works slightly differently.